contrib/exporters/secadvisor/mod/acc_rej_action.go

```diff
@@ -0,0 +1,218 @@
+				// got duplicate from same flow; need to indicate reject
+				// or accumulate the stats until next time
+				flowInf.Saw1 = true
+				flowInf.Flow1 = f
+				continue
+			}
+		} else { // normal case
+			if flowInf.UUID1 == f.UUID {
+				flowInf.Flow1 = f
+				flowInf.Saw1 = true
+			} else {
+				flowInf.Flow2 = f
+				flowInf.Saw2 = true
+			}
+		}
+	}
+}
+
+// combineFlows - helper function to combine 2 seen flows into one, based on common minimums
+func combineFlows(f *accRejFlowInfo) *flow.Flow {
+	// perform shallow copy of flow and then update some fields
+	newF := *f.Flow1
+	var newMetricReported flow.FlowMetric
+	var newLastUpdateMetric flow.FlowMetric
+	// Note: Metric.Start should never change
+	newMetricReported.Start = f.Flow1.Metric.Start
+	// take byte and packet count as minimum of the 2 reports
+	newMetricReported.ABPackets = common.MinInt64(f.Flow1.Metric.ABPackets, f.Flow2.Metric.ABPackets)
+	newMetricReported.BAPackets = common.MinInt64(f.Flow1.Metric.BAPackets, f.Flow2.Metric.BAPackets)
+	newMetricReported.ABBytes = common.MinInt64(f.Flow1.Metric.ABBytes, f.Flow2.Metric.ABBytes)
+	newMetricReported.BABytes = common.MinInt64(f.Flow1.Metric.BABytes, f.Flow2.Metric.BABytes)
+
+	// if we are at the end of the flow, set Last time to latest of the times measured
+	if f.Flow1.FinishType != flow.FlowFinishType_NOT_FINISHED && f.Flow2.FinishType != flow.FlowFinishType_NOT_FINISHED {
+		newMetricReported.Last = common.MaxInt64(f.Flow1.Metric.Last, f.Flow2.Metric.Last)
+	} else {
+		newMetricReported.Last = common.MinInt64(f.Flow1.Metric.Last, f.Flow2.Metric.Last)
+	}
+
+	// compute LastUpdateMetric based on previous MetricReported and current newMetricReported
+	if f.Status == FlowInfoOldFlow {
+		newLastUpdateMetric.ABPackets = newMetricReported.ABPackets - f.MetricReported.ABPackets
+		newLastUpdateMetric.ABBytes = newMetricReported.ABBytes - f.MetricReported.ABBytes
+		newLastUpdateMetric.BAPackets = newMetricReported.BAPackets - f.MetricReported.BAPackets
+		newLastUpdateMetric.BABytes = newMetricReported.BABytes - f.MetricReported.BABytes
+		newLastUpdateMetric.Start = f.MetricReported.Last
+	} else {
+		// this is the first report for the flow
+		newLastUpdateMetric.ABPackets = common.MinInt64(f.Flow1.LastUpdateMetric.ABPackets, f.Flow2.LastUpdateMetric.ABPackets)
+		newLastUpdateMetric.BAPackets = common.MinInt64(f.Flow1.LastUpdateMetric.BAPackets, f.Flow2.LastUpdateMetric.BAPackets)
+		newLastUpdateMetric.ABBytes = common.MinInt64(f.Flow1.LastUpdateMetric.ABBytes, f.Flow2.LastUpdateMetric.ABBytes)
+		newLastUpdateMetric.BABytes = common.MinInt64(f.Flow1.LastUpdateMetric.BABytes, f.Flow2.LastUpdateMetric.BABytes)
+		newLastUpdateMetric.Start = newMetricReported.Start
+	}
+	newLastUpdateMetric.Last = newMetricReported.Last
+
+	newF.Metric = &newMetricReported
+	newF.LastUpdateMetric = &newLastUpdateMetric
+
+	// FIXME - What should we do with UpdateCount? In the meantime, we have the value from Flow1
+	newF.Action = "ACCEPT"
```

Comment:
the strings for ACCEPT and REJECT are already defined in `contrib/exporters/awsflowlogs/mod` - please reuse those definitions.

Reply:
This would require renaming of numerous items to make these items available globally. For now we keep ACCEPT and REJECT as is. This can be refactored together with the aws code in a separate PR.